README.md

```diff
@@ -1,2 +1,115 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
```

Comment:
grammar: 
The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components.

TAU **components** allow to create and manage **components**?
or TAU contains components...

Reply:
agreed version: The TAU library contains components which allows you to create Web Application UI. Each components represents UI element, such as a...,